docgen/src/examples/e-commerce-infinite/App.js

```diff
@@ -160,7 +120,8 @@ const ColorItem = ({item, createURL, refine}) => {
 };
 
 const CustomColorRefinementList = ({items, refine, createURL}) =>
-    <div>
+    items.length > 0 ? <div>
+      <h5 className={'ais-Panel__Title'}>Colors</h5>
```

Comment:
connectors could forward the isEmpty function down the line when used in standalone?

That's maybe too much and in the end people using connectors may be always be using connectors, not mixing widgets like we do in examples.